packages/heroic-games-launcher-deb/heroic-games-launcher-deb.pacscript

```diff
@@ -1,8 +1,8 @@
 name="heroic-games-launcher-deb"
 gives="heroic"
-repology=("project: ${gives}")
-version="2.5.2"
+repology=("project: heroic-games-launcher")
```

Comment:
Why did you change the repology project?

Reply:
This repology gives latest updates